docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

```diff
@@ -3,67 +3,129 @@ title: Upgrade Expo SDK
 ---
 
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
 > **info** We recommend upgrading SDK versions incrementally, one at a time. Doing so will help you pinpoint breakages and issues that arise during the upgrade process.
 
-Expo maintains ~6 months of backward compatibility. Once an SDK version has been deprecated, you will no longer be able to use the Expo Go app. However, you will still be able to publish updates via `eas update`, and build using `eas build`. Deprecations **will not** affect the standalone apps you have in production.
+Expo maintains ~6 months of backward compatibility. Once an SDK version has been deprecated, you will no longer be able to use the Expo Go app. However, you will still be able to publish updates via `eas update`, and build using `eas build`. Deprecations **will not** affect the standalone apps you have in production or your [development builds](/develop/development-builds/introduction/).
+
+## How to upgrade to the latest SDK version
+
+<Step label="1">
+
+### Upgrade EAS CLI
+
+Update to the latest version of EAS CLI (if you use it):
+
+<Terminal cmd={['$ npm i -g eas-cli']} />
+
+</Step>
+
+<Step label="2">
+
+### Upgrade the Expo SDK
+
+Install the new version of the Expo package:
+
+<Tabs>
+<Tab label="npm">
+
+<Terminal cmd={['$ npm install expo@latest']} />
+
+</Tab>
+
+<Tab label="yarn">
+
+<Terminal cmd={['$ yarn install expo@latest']} />
+
+</Tab>
+</Tabs>
+
+</Step>
+
+<Step label="3">
+
+### Upgrade dependencies
+
+Upgrade all dependencies to match SDK 49:
+
+<Terminal cmd={['$ npx expo install --fix']} />
+
+</Step>
+
+<Step label="4">
+
+### Follow the release notes for any other instructions
+
+Read the [SDK changelogs](#sdk-changelogs) for the SDK version you are upgrading to. They contain important information about breaking changes, deprecations, and other changes that may affect your app.
+
+</Step>
+
+## Upgrading native projects
+If you manage your own native projects (bare workflow), you will need to manually apply any changes to your native Android and iOS project folders. Our [bare upgrade advisor](/bare/upgrade) will show you a diff that describes the changes you need to make. Alternatively, you could consider [adopting prebuild](/guides/adopting-prebuild) for easier upgrades in the future.
+
+## SDK Changelogs
+Each SDK announcement blog post contains deprecations, breaking changes, and anything else that might be unique to that particular SDK version. When upgrading, be sure to check these out to make sure you don't miss anything.
 
-## SDK 49
+### SDK 49
 
 [Blog Post](https://blog.expo.dev/expo-sdk-49-c6d398cdf740)
 
-## SDK 48
+### SDK 48
 
 [Blog Post](https://blog.expo.dev/expo-sdk-48-ccb8302e231)
 
-## SDK 47
+### SDK 47
 
 [Blog Post](https://blog.expo.dev/expo-sdk-47-a0f6f5c038af)
 
-## SDK 46 [DEPRECATED]
+### SDK 46 [DEPRECATED]
```

Comment:
can we move all deprecated changelogs under a heading called Deprecated SDK Versions and then remove the [DEPRECATED] text from the title and change to a heading such as `####` that won't be included in the sidebar?